recipe/meta.yaml

```diff
@@ -1,9 +1,7 @@
-{%set name = "Flask-Creole" %}
-{%set version = "0.4.4" %}
-{%set compress_type = "tar.gz" %}
-{%set hash_type = "sha256" %}
-{%set hash_val = "af308ba55fdda617d715877d74cdedea75727669e0555ef2a09539220318622d" %}
-{%set build_num = "0" %}
+{% set name = "Flask-Creole" %}
+{% set version = "0.4.4" %}
+{% set compress_type = "tar.gz" %}
```

Comment:
I would get rid of all the unnecessary `jinja`. Most of these are just silly or useless now that we have the bot. IMO only `version` is still justified and maybe `name`.